docs/opentelemetry.md

```diff
@@ -1,8 +1,8 @@
 # OpenTelemetry
 
-This SDK produces [metrics](https://opentelemetry.io/docs/concepts/signals/metrics/) using [OpenTelemetry](https://opentelemetry.io/) that allow you to view data such as request timings. These metrics also include attributes for the model and store ID, as well as the API called to allow you to build reporting.
+This SDK produces [metrics](https://opentelemetry.io/docs/concepts/signals/metrics/) using [OpenTelemetry](https://opentelemetry.io/) to allow you to view data such as request timings. These metrics also include attributes for the model and store ID and the API called to allow you to build reporting.
```

Comment:
I'm not sure whether removing the multiple `and`'s works better, what do you think?

```suggestion
This SDK produces [metrics](https://opentelemetry.io/docs/concepts/signals/metrics/) using [OpenTelemetry](https://opentelemetry.io/) to allow you to view data such as request timings. These metrics also include attributes for the model ID, store ID and the API called to allow you to build reporting.
```

Reply:
I think using and's seems redundant.